docs/overview/what-is-ern.md

```diff
@@ -74,4 +74,4 @@ An Electrode Native runner application is automatically generated the first time
 
 [Learn More about the Runner](../platform-parts/runner/index.md)
 
-[Electrode Native Bridge]:https://github.com/electrode-io/react-native-electrode-bridge
\ No newline at end of file
+[Electrode Native Bridge]:https://github.com/electrode-io/react-native-electrode-bridge
```

Comment:
What changed here❓

Reply:
https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
So weird.. I was pretty sure I commented on the last line in `bundlestore.md` - The removed line and the added line look exactly the same (here on GitHub)...

Reply:
Checking the commit content on command line revealed that you added the missing newline at end of file 👍